diagrams/request_uri_mode_create.md

```diff
@@ -0,0 +1,49 @@
+```plantuml
+@startuml
+
+autonumber
+
+box "Wallet"
+participant "Metadata" as wm
+participant "Authorization Endpoint" as w
+end box
+
+participant "User Agent" as u
+
+box "Verifier"
+participant "Frontend" as r
+participant "Request Endpoint" as rp
+participant "Response Endpoint" as rb
+end box
+
+u --> r : use
+activate r
+
+r --> u: **signed authorization request**\n(client_id, request_uri, request_uri_mode=create, state)
+deactivate r
+u --> w: **signed authorization request**\n(client_id, request_uri, request_uri_mode=create, state)
+activate w
+w --> w: check authorization request signature
```

Comment:
5. We need to add similar language as described in Connect/JAR that says that if the `request_uri` Request Object contains the same parameters as in the initial Authz Request, the `request_uri` Request Object has priority.

Reply:
I guess we need to add this in the spec text? Please add the respective comment(s) where you would like to see it added.

Reply:
It seems quite normative to me and this comment/request4change should be moved in the normative text and not in this sequence diagram example

do you agree @awoie ?

Reply:
added requested normative text

https://github.com/openid/OpenID4VP/pull/59/commits/acd0c135d3cdcbb485286b0911cb1a17a5298f68

Reply:
```suggestion
The Wallet MUST extract the set of authorization request parameters from the Request Object returned from the Verifier's request URI. The Wallet MUST only use the parameters in this Request Object, even if the same parameter was provided in an authorization request query parameter or a request object passed in the authorization request through the `request` parameter. The Client ID value in the `client_id` authorization request parameter (or `request` object claim) and the `client_id` claim in the Request Object obtained from the Request URI MUST be identical. The Wallet then validates the request, as specified in OAuth 2.0 [RFC6749].
```

IMO, it is important that the `client_id` from the authz request matches the `client_id` from the Request URI endpoint. The requirement that `client_id` from `request` parameter must match `client_id` from Authz Request is already defined in RFC9101.